aips/aip-lite-account.md

```diff
@@ -0,0 +1,143 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: Introduce Lite Account (Account v2)
+author: lightmark, davidiw
+Status: Draft
+type: Framework
+created: <06/25/2024>
+---
+
+# AIP-X - Lite Account, aka Account V2
```

Comment:
This should match what is in `title` above.

Also, can we name this one thing, not two things?
It's either **lite accounts** or `account_v2`, but please don't call it both.

Reply:
+1